Mathlib/RingTheory/Flat/CategoryTheory.lean

```diff
@@ -6,6 +6,9 @@ Authors: Jujian Zhang
 import Mathlib.RingTheory.Flat.Basic
 import Mathlib.Algebra.Homology.ShortComplex.ModuleCat
 import Mathlib.Algebra.Category.ModuleCat.Monoidal.Basic
+import Mathlib.Algebra.Category.ModuleCat.Projective
```

Comment:
Can you move this iso to a new file to avoid import creep here? Perhaps add an appropriate `assert_not_exists` here to guard against future imports of `Tor` here.